manifests/00-crd.yaml

```diff
@@ -2,7 +2,6 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    include.release.openshift.io/ibm-cloud-managed: "true"
```

Comment:
@joelddiaz if we're not removing cloud cred requests from all repos before 4.10, then it doesn't make sense to exclude the CRD just yet

Reply:
Yes, I was thinking we'd block this PR until all the depending PRs merge (which of course means we're looking at this stuff landing for 4.11).